extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/StepUpAuthenticationPolicy.java

```diff
@@ -147,4 +151,13 @@ static void markAsEnabled() {
     static boolean isEnabled() {
         return enabled;
     }
+
+    private static String[] splitAcrValues(String acrValues) {
+        String[] acrValuesArr = acrValues.split(",");
+        if (acrValuesArr.length > 1) {
+            // it doesn't make sense, but it is theoretically possible that user will specify one value multiple times
```

Comment:
```suggestion
```
This is a sound code removing duplicates so I believe we don't have to explain it 🙂, may be we could instead warn but I'd not be contacted